app/workers/ip_location_worker.rb

```diff
@@ -0,0 +1,13 @@
+class IpLocationWorker
+  include Sidekiq::Worker
+
+  def perform(id, ip_address)
+    @user = User.find(id)
+    location = Pointpin.locate(ip_address)
+    unless location["postcode"] == ("10005" || "11237" || "11385") #locations where we've had offices and therefore triggered a ton of false ip addresses
```

Comment:
Do we really need to set this on every sign in? We dont need to black list IPs if if is only set on creating the user